examples/apps/data-object-grid/package.json

```diff
@@ -110,28 +110,6 @@
 			}
 		}
 	},
-	"fluidBuild": {
-		"tasks": {
-			"eslint": [
-				"^tsc",
-				"@fluid-example/clicker#build:esnext",
-				"@fluid-example/codemirror#build:esnext",
-				"@fluid-example/collaborative-textarea#build:esnext",
-				"@fluid-example/multiview-coordinate-model#build:esnext",
-				"@fluid-example/multiview-slider-coordinate-view#build:esnext",
-				"@fluid-example/prosemirror#build:esnext"
-			],
-			"build:esnext": [
-				"^tsc",
-				"@fluid-example/clicker#build:esnext",
-				"@fluid-example/codemirror#build:esnext",
-				"@fluid-example/collaborative-textarea#build:esnext",
-				"@fluid-example/multiview-coordinate-model#build:esnext",
-				"@fluid-example/multiview-slider-coordinate-view#build:esnext",
-				"@fluid-example/prosemirror#build:esnext"
-			]
-		}
-	},
```

Comment:
Why are these no longer needed?

Reply:
They were too limited. There were missing things and now are covered by the base config.
These changes found the hole - this is why we want them.

Reply:
I confirmed that these all seem covered by the base config now.